packages/controller-utils/src/siwe.test.ts

```diff
@@ -0,0 +1,28 @@
+import { ParsedMessage } from '@spruceid/siwe-parser';
+import { detectSIWE } from './siwe';
+
+const mockedParsedMessage = {
+  domain: 'example.eth',
+  address: '0x0000000',
+};
+
+jest.mock('@spruceid/siwe-parser');
+
+describe('detectSIWE', () => {
+  const parsedMessageMock = ParsedMessage as any;
+  it('should return an object with isSIWEMessage set to true and parsedMessage', () => {
```

Comment:
Nit: lately the description format we've been using is more like:

```suggestion
  it('returns an object with isSIWEMessage set to true and parsedMessage', () => {
```

i.e. drop the "should", and have the description describe what the thing-under-test _does_ in this scenario.

Apologies for not having clear docs for this yet - those will be coming soon though.